SU-HSDO/suhumsci
# fix(STN-489)  .hs-well

# READY FOR REVIEW

## Summary
Fixes hs-well for when it appears in the second sidebar for three column layouts. Also switches backstop json to test the staging site.

**Browser Testing**

## Steps to Test
1. `npm run test` passes in humci_basic.
2. Go to http://sparkbox-sandbox.suhumsci.loc/people/ran-abramitzky and ensure that for a block in the second sidebar with the class hs-well, it goes 100% of that sidebar width. Compare to the bug here: https://sparkbox-sandbox-stage.stanford.edu/people/ran-abramitzky
3. Ensure nothing has changed on this page: http://sparkbox-sandbox.suhumsci.loc/breakout-and-quote-styles (compare to stage)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
